Library/Formula/sagittarius-scheme.rb

```diff
@@ -0,0 +1,33 @@
+class SagittariusScheme < Formula
+  desc "Free Scheme implementation supporting R6RS and R7RS"
+  homepage "https://bitbucket.org/ktakashi/sagittarius-scheme/wiki/Home"
+  url "https://bitbucket.org/ktakashi/sagittarius-scheme/downloads/sagittarius-0.7.0.tar.gz"
+  sha256 "365ccb0c0779717b04111ea86962d24a4841c07289426003f24782295fbe4658"
+  head "https://bitbucket.org/ktakashi/sagittarius-scheme", :using => :hg
+
+  option "with-docs", "Build HTML docs"
```

Comment:
Unless the docs are ridiculously large we prefer to not make people recompile from source to obtain them.